source/gui/_addonStoreGui/controls/storeDialog.py

```diff
@@ -278,6 +294,15 @@ def _channelFilterKey(self) -> Channel:
 	def _titleText(self) -> str:
 		return f"{self.title} - {self._statusFilterKey.displayString} ({self._channelFilterKey.displayString})"
 
+	_warningText = pgettext(

```

Comment:
As the warning text is a multiline text control, NVDA only reads the first line when it is focused.
On my system I only hear:
"Add-ons are created by the NVDA community and are not vetted by NV Access. NV Access cannot be held responsible for add-on behavior. The functionality of add-ons is"
Of course I can then press down arrow to hear the rest.
I wonder if we can get away with a much shorter to the point message which can be read in one line such as:
"Add-ons are provided by the community and not vetted or maintained by NV Access. Add-ons could make your system insecure or unstable. Install at your own risk."
This is 157 characters which fits on one line on my system at least.
We could revisit the message when we have more time to come up with a better control which can have the entire message read on focus.